sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Customizations/Models/AssetConversionConfiguration.cs

```diff
@@ -44,6 +45,28 @@ internal AssetConversionConfiguration(Vector3 gravityWrapper, float scale)
             KeyFrameIndexes = new ChangeTrackingList<int>();
             GroundTruthTrajectoryCameraPoses = new List<TrajectoryPose>();
             Scale = scale;
+            DisableDetectScaleUnits = false;
```

Comment:
This is the constructor with (gravity, scale) params, we do not have disableDetectScaleUnits passed in. This constructor is used to avoid generating same constructor in the generated code which causes error.